config/settings/base.py

```diff
@@ -72,6 +72,7 @@
     "allauth",
     "allauth.account",
     "allauth.socialaccount",
+    "commcare_connect.commcarehq_provider",
```

Comment:
this should go in `LOCAL_APPS`

Reply:
Fixed. [57067aa](https://github.com/dimagi/commcare-connect/pull/13/commits/57067aa21ede4d53afeb323eacaf20e11f3b9181)